Makefile

```diff
@@ -25,24 +37,37 @@ lint:
 .PHONY: binary
 binary:
 	@echo "WARNING: binary creates a Linux executable. Use cross for macOS or Windows."
+	# TODO(stevvooe): Remove the use of this script. It does nothing. It is
+	# more complicated because the variables need to be converted back from
+	# shell.
 	./scripts/build/binary
 
 .PHONY: cross
 cross:
+	# TODO(stevvooe): Fix the cross build so this isn't a separate target.
+	# Should simply be able to do `make GOOS=windows` to get a cross-compiled
+	# binary.
 	./scripts/build/cross
 
 .PHONY: dynbinary
 dynbinary:
+	# TODO(stevvooe): Change this to target binary but enable cgo, as that is
+	# the only difference.
 	./scripts/build/dynbinary
 
-.PHONY: watch
-watch:
-	./scripts/test/watch
-
 # Check vendor matches vendor.conf
 vendor: vendor.conf
 	vndr 2> /dev/null
-	scripts/validate/check-git-diff vendor
+	if [ "`git status --porcelain -- vendor/ 2>/dev/null`" ]; then \
```

Comment:
Ah yes, new files. I think a `git add -N vendor` prior to the `git diff --exit-status` (whichever variant) would make that work as desired, but I've not actually tried it. In particular I'm not sure what the effect on removed files will be.